fitbenchmarking/results_processing/local_min_table.py

```diff
@@ -102,23 +101,11 @@ def get_value(self, result):
                                       x=result.data_x,
                                       y=result.data_y,
                                       e=result.data_e)
-        try:
-            jac = result.cost_func.jac_res(result.params,
-                                           x=result.data_x,
-                                           y=result.data_y,
-                                           e=result.data_e)
-        except NotImplementedError:
-            # If using a solver dependent Jacobian, we want
-            # to switch to a common jacobian here to generate
-            # comparisons
-            jacobian_cls = create_jacobian("scipy")
-            jacobian_instance = jacobian_cls(result.cost_func.problem)
-            jacobian_instance.method = "2-point"
-            result.cost_func.jacobian = jacobian_instance
-            jac = result.cost_func.jac_res(result.params,
-                                           x=result.data_x,
-                                           y=result.data_y,
-                                           e=result.data_e)
+
+        jac = result.cost_func.jac_res(result.params,
+                                       x=result.data_x,
+                                       y=result.data_y,
+                                       e=result.data_e)
```

Comment:
Of course -- the default is now scipy-2 point, and so there's no need for an if statement....much neater!